### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by Hightech1011** [roxlukas/lmeve#25]
After completing the install and trying to login for the first time, I get the following error:

Wrong username
or password.

I have run "php ./bin/passwd.php", and got the following:

User 'admin' already exists. New password is 'admin'
Login to LMeve, go to Settings and change admin password immediately

I still get the wrong unsername/password error. I even dropped the used from the "lmuser" table (DELETE FROM `lmusers` WHERE `login`='admin';) and reran "passwd.php" and got:

User 'admin' does not exist. Adding user 'admin' and setting password 'admin'
Adding 'Administrator' role to user 'admin'
Login to LMeve, go to Settings and change admin password immediately

But still unable to login.

Did I miss something?

**Comment by roxlukas** [comment]
I missed updating db schema after adding sha256 password hashes. Please run this command in MySQL:

```
ALTER TABLE  `lmusers` CHANGE  `pass`  `pass` VARCHAR( 64 );
```

Then simply re-run the password reset procedure.

**Comment by roxlukas** [comment]
Fixed in 35295c7
